cloudinit/sources/DataSourceLXD.py

```diff
@@ -14,7 +14,7 @@
 import stat
 from enum import Flag, auto
 from json.decoder import JSONDecodeError
-from typing import Any, Dict, Union, cast
```

Comment:
I think you can also remove the `TODO ( Hotplug support using websockets API) comment in the docstr too.